ledger-core/v2/virtual/execute/execute.go

```diff
@@ -116,9 +116,15 @@ func (s *SMExecute) prepareExecution(ctx context.Context) {
 	}
 }
 
+func (s *SMExecute) defaultMigration(ctx smachine.MigrationContext) smachine.StateUpdate {
```

Comment:
I suggest to rename it to "migrationDefault"

Reply:
done